docs/en/internationalization.md

```diff
@@ -37,7 +36,7 @@ return array(
 		'about' => 'À propos de FreshRSS',
 	),
 	// ...
-];
+);
```

Comment:
@Alkarex Please check the syntax of this array, especially the brackets in line 27 and 33

Reply:
Indeed, line 27 is wrong

Reply:
fixed in #5392